### PR TITLE
Latest build

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -18,7 +18,8 @@ dependencies:
   meta: ^1.0.5
 
 dev_dependencies:
-  build_runner: '>=0.4.0 <0.7.0'
+  build_config: '^0.2.1'
```
Comment (natebosch) [inline pubspec.yaml]: why do you need this dep?

**Comment by kevmoo** [comment]
InputSet -> `generateFor: const InputSet(include: const
['lib/src/specs/**.dart']),`

worth exporting?

**Comment by natebosch** [comment]
> worth exporting?

Not sure. I guess this is probably enough of an edge case that it won't matter much. Once we update built_value to expose a build.yaml this file will go away.

**Comment by matanlurey** [comment]
Let's hold off here and wait until we can get `build.yaml` working.

**Pull request closed without merging**